test/integration/utils/wait_for_re_api.py

```diff
@@ -0,0 +1,45 @@
+"""
+Block until all dependent services come online.
```

Comment:
is there no way to integrate this with the existing `wait_for` code?

Reply:
Yes, of course; I broke it out because I needed changes (e.g. utilize RE_API_URL), and the original one handles more cases than integration tests need.

Reply:
It'd be far preferable not to have functionality duplicated in different places in the codebase -- it's more code to maintain, more potential for bit rot and cruft to build up, etc.